### PR TITLE
Update messages.json for HK

### PR DESCRIPTION
**Pull request by discord1024**
change mobile wording for HK

### CODE REVIEW

File: messages.json [hunk 1]
```diff
@@ -583,11 +583,23 @@
...
       "zh_HK": {
-         "purchase.title": "蓝灯专业版",
-         "purchase.subtitle": "速度更快。 更稳定。无流量限制。免费加密货币隐币",
-         "purchase.button_text": "立即升级",
-         "purchase.reseller_prompt": "网上打折购买了专业版激活码？",
-         "purchase.reseller_link_text": "点这里使用激活码升级，免费开抢隐币",
+         "purchase.title": "藍燈 無限流量VPN",
+         "purchase.subtitle": "不儲存任何信息 不予港警合作 忽略香港法庭命令 境外服務器",
+         "purchase.button_text": "香港用戶免費",
+         "purchase.reseller_prompt": "打開藍燈安全隱秘的使用互聯網！",
```
Comment (joesis) [inline messages.json:589]: Not a big deal, but 的 should be 地 I guess, unless things are different in Cantonese

**Comment by joesis** [comment]
Cool @discord1024 merging!

**Comment by joesis** [comment]
Oh well it won't work as expected @discord1024 as at least on desktop, the traditional Chinese option is `zh-TW` rather than `zh-HK`...

**Comment by discord1024** [comment]
Desktop is take care of already with desktop-UI and verified to work.

**Comment by joesis** [comment]
Okay cool, I can not see anything relevant in https://github.com/getlantern/lantern-desktop-ui/pulls?q=is%3Apr though.

**Comment by discord1024** [comment]
sorry I meant https://github.com/getlantern/loconf/blob/master/desktop-ui.json

**Comment by joesis** [comment]
That's interesting @discord1024 so Lantern shows the HK messages when you set OS location and language accordingly?  Anyway I made https://github.com/getlantern/loconf/pull/28 to show HK messages for zh-TW as well, because if you manually change the language to "traditional Chinese", the internal representation is zh-TW.

**Comment by discord1024** [comment]
@Alexatall made a good point that maybe we want to make it hard for users to see the message, so mainland users don't get pissed off. So perhaps it's better so users cannot manually switch and see the language? @Alexatall what do you think?

**Pull request merged**